src/main/modules/helmet/index.ts

```diff
@@ -31,7 +31,7 @@ export class Helmet {
           fontSrc: [self, 'data:'],
           imgSrc: [self, googleAnalyticsDomain],
           objectSrc: [self],
-          scriptSrc: [self, googleAnalyticsDomain, "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"],
+          scriptSrc: [self, googleAnalyticsDomain, "'unsafe-inline'", "'unsafe-eval'"],
```

Comment:
Any chance you could explain this?

Reply:
At this end of the template.njk, there is an inline script which I copied from the FaCT project. This doesn't satisfy the security policy unless I change this policy. Strangely it now works when I change it back to original. Not sure why but will keep an eye out for it